Source/Core/DolphinQt/Config/MouseSettingsWidget.cpp

```diff
@@ -0,0 +1,142 @@
+
+#include <memory>
+
+#include <QApplication>
+#include <QCheckBox>
+#include <QDoubleSpinBox>
+#include <QLabel>
+#include <QVBoxLayout>
+
+#include "Core/Config/MouseSettings.h"
+#include "DolphinQt/QtUtils/MultiClickSignalButton.h"
+#include "InputCommon/ControllerInterface/OctagonalMouseJail.h"
+
+MouseSettingsWidget::MouseSettingsWidget(QWidget* parent) : QGroupBox(tr("Mouse"), parent)
+{
+  CreateLayout();
+  LoadSettings();
+  ConnectWidgets();
+}
+
+void MouseSettingsWidget::CreateLayout()
+{
+  QVBoxLayout* primary_layout = new QVBoxLayout{};
+
+  QHBoxLayout* sensitivity_layout = CreateSensitivityLayout();
+  QHBoxLayout* snapping_distance_layout = CreateSnappingDistanceLayout();
+
+  QHBoxLayout* octagonal_mouse_jail_is_enabled = CreateOctagonalMouseJailIsEnabledLayout();
+
+  primary_layout->addLayout(octagonal_mouse_jail_is_enabled);
+  primary_layout->addLayout(sensitivity_layout);
+  primary_layout->addLayout(snapping_distance_layout);
+
+  this->setLayout(primary_layout);
+}
+
+void MouseSettingsWidget::ConnectWidgets()
+{
+  connect(m_sensitivity_control, QOverload<double>::of(&QDoubleSpinBox::valueChanged), this,
+          &MouseSettingsWidget::SaveSettings);
+  connect(m_snapping_distance_control, QOverload<double>::of(&QDoubleSpinBox::valueChanged), this,
+          &MouseSettingsWidget::SaveSettings);
+  connect(m_octagonal_mouse_jail_is_enabled_control, &QCheckBox::clicked, this,
+          &MouseSettingsWidget::SaveSettings);
+}
+
+void MouseSettingsWidget::LoadSettings()
+{
+  m_sensitivity_control->setValue(Config::Get(Config::MOUSE_SENSITIVITY));
+  m_snapping_distance_control->setValue(Config::Get(Config::MOUSE_SNAPPING_DISTANCE));
+  m_octagonal_mouse_jail_is_enabled_control->setChecked(
+      Config::Get(Config::MOUSE_OCTAGONAL_JAIL_ENABLED));
+}
+
+void MouseSettingsWidget::SaveSettings()
+{
+  Config::SetBaseOrCurrent(Config::MOUSE_SENSITIVITY, m_sensitivity_control->value());
+  Config::SetBaseOrCurrent(Config::MOUSE_SNAPPING_DISTANCE, m_snapping_distance_control->value());
+  Config::SetBaseOrCurrent(Config::MOUSE_OCTAGONAL_JAIL_ENABLED,
+                           m_octagonal_mouse_jail_is_enabled_control->isChecked());
+  Config::Save();
+  ciface::OctagonalMouseJail::GetInstance().RefreshConfigValues();
+}
+
+QHBoxLayout* MouseSettingsWidget::CreateSensitivityLayout()
+{
+  QHBoxLayout* sensitivity_layout = new QHBoxLayout{};
+
+  m_sensitivity_control = new QDoubleSpinBox{};
+  m_sensitivity_control->setRange(1.00, 9999.50);
+  m_sensitivity_control->setDecimals(2);
+  m_sensitivity_control->setSingleStep(0.5);
+  m_sensitivity_control->setWrapping(true);
+  QString sensitivity_tool_tip{
+      tr("Adjusts how quickly the mouse cursor moves the Cursor X/Y +/- inputs."
+         "\nChanges the distance at which your mouse snaps to notches. May make your snapping feel "
+         "weird."
+         "\n\n1.0(default) sensitivity maps the mouse to the full screen"
+         "\n\nRecommended value: 7.5")};
```

Comment:
I think you placed the description of "Notch Distance" here.
But anyway, we can assume users are familiarized with concept of mouse sensitivity. Indeed, many mouses can configure that through included software or some switch in the hardware. It's important to inform the user how the setting will interact with those external settings (will it be overridden or compounded).
As for the setting description, I would go with:
```
Adjusts how much the mouse, moves the cursor around under a given input.
The recommended value is 7.5
```